extensions/binding/org.eclipse.smarthome.binding.dmx/src/main/java/org/eclipse/smarthome/binding/dmx/internal/dmxoverethernet/DmxOverEthernetHandler.java

```diff
@@ -108,7 +108,7 @@ protected void sendDmxData() {
                     try {
                         socket.send(sendPacket);
                     } catch (IOException e) {
-                        logger.debug("Could not send to {} in {}: {}", receiverNode, this.thing.getUID(), e);
+                        logger.debug("Could not send to {} in {}: ", receiverNode, this.thing.getUID(), e);
```

Comment:
since this is debug logging provide an `e.getMessage()` instead and re-add the placeholder.